config/sync/core.entity_view_display.node.stanford_opportunity.default.yml

```diff
@@ -26,25 +26,27 @@ dependencies:
     - field.field.node.stanford_opportunity.su_opp_tags
     - field.field.node.stanford_opportunity.su_opp_type
     - node.type.stanford_opportunity
+    - system.menu.main
```

Comment:
On all our other content types, we display the title above the rest of the columns. Should we keep the same behavior?
![image](https://github.com/user-attachments/assets/080f3140-5190-4f1c-aaff-9c21a67c773e)